testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestReadUniontype.java

```diff
@@ -270,6 +271,53 @@ private void testAvroSchemaEvolution()
         }
     }
 
+    @Test(dataProvider = "storage_formats", groups = SMOKE)
+    public void testReadUnionAsStruct(String storageFormat)
```

Comment:
Can we merge this into existing testUnionTypeSchemaEvolution ? Or atleast call this something explicit like testUnionToStructSchemaEvolution

Reply:
done.